### ISSUE TITLE
Error when try to generate a Bundle with a User

### ISSUE THREAD

**Issue by freddyucv** [dotCMS/core#22508]
**Describe the bug**
When we go try to generate a bundle that contains any User we are getting this error

```
Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make field private final jdk.internal.loader.BuiltinClassLoader jdk.internal.loader.BuiltinClassLoader.parent accessible: module java.base does not "opens jdk.internal.loader" to unnamed module @fadd4d07
	at java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:340) ~[?:?]
	at java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:280) ~[?:?]
	at java.lang.reflect.Field.checkCanSetAccessible(Field.java:176) ~[?:?]
	at java.lang.reflect.Field.setAccessible(Field.java:170) ~[?:?]
	at com.thoughtworks.xstream.converters.reflection.FieldDictionary.buildMap(FieldDictionary.java:148) ~[xstream-1.4.8.jar:1.4.8]
```

**To Reproduce**
1. Go to User portlet
2. Select the "Add to Bundle" option
3. Go to 'Publishing Queue' portlet and into the Bundle tab, select download Bundle

**Expected behavior**
Should generated the bundle without problem

**Comment by dsilvam** [comment]
Passed Internal QA: Bundle with all users generating correctly.

**Comment by bryanboza** [comment]
Fixed, tested on release-22.07 // Docker // FF. I'm unable to reproduce after this fix

**Comment by erickgonzalez** [comment]
Not back portable to 21.06 LTS due to changes in Manifest.
